src/components/eRACommons.js

```diff
@@ -120,7 +120,7 @@ export const eRACommons = hh(class eRACommons extends React.Component {
       width: 38,
       backgroundRepeat: 'no-repeat',
       backgroundSize: 'contain',
-      backgroundImage: 'url("/images/era-commons-logo.png")',
+      backgroundImage: `url(${eraIcon})`,
```

Comment:
Even though we're defining a css rule here, because we're in a js file we'll have to use import references and template strings to get the desired result.